docs/ios/v2/plugins/local-camera-capture.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Camera Frame Capture Plugin (Beta)
+nav: 5.3
+---
+
+## Introduction
+
+You can use Camera Frame Capture Plugin to capture the current frame from the local camera in hight quality. This can be useful if you want to implement a remote KYC photo capture use-case where an agent can capture a high quality photo from the client device.
```

Comment:
Whats a remote KYC?

Reply:
It's KYC stands for know-your-customer. This is what Navi is doing. It's a well known use case here in India. Companies require to do this capturing photos of client's original documents. Now they do this remotely. I will add the full form.